lib/ex_unit/lib/ex_unit.ex

```diff
@@ -428,6 +428,30 @@ defmodule ExUnit do
     end
   end
 
+  def mark_files_as_failed(modules, allowed_filenames, manifest_filename) do
```

Comment:
@josevalim This code is currently working as expected through manual tests (I created a local project using the compiled mix/ex_unit versions).

Any pointers on how we could test this? Or is this even the proper way to do it?

Reply:
There's a failing test which I suspect is what needs to be changed, but I'm not sure if it breaking is a bug or indeed just a consequence of the current change.